src/main/scala/com/horizon/exchangeapi/ManagementPoliciesRoutes.scala

```diff
@@ -181,13 +181,13 @@ trait ManagementPoliciesRoutes extends JacksonSupport with AuthenticationSupport
   } // end of exchAuth
   }
 
-  /* ====== GET /orgs/{orgid}/managementpolicies{policy} ================================ */
+  /* ====== GET /orgs/{orgid}/managementpolicies/{nmpid} ================================ */
```

Comment:
I think we should use `{mgmtpolicy}` instead of `{nmpid}` for consistency purposes. Similar to how it is used in NodeRoutes.scala for the NodeMgmtPolicy status routes